docs/equipment/artifacts.md

```diff
@@ -380,6 +380,8 @@ import Artifact from '@site/src/components/artifact/Artifact'
 
 <Artifact artifact="Adventurer"/>
```

Comment:
Need to expand this node so the finding is inside the artifact blob

Reply:
go over it once but should be fine now